File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0) return false;
        string s=to_string(x);
        int length=s.size();
        for(int i=0;i<length/2;i++){
            if(s[i]!=s[length-1-i]) return false;
        }
        return true;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0) return false;
        String xstring=Integer.toString(x);

        int len=xstring.length();
        for(int i=0;i<len;i++){
            if(xstring.charAt(i)!=xstring.charAt(len-i-1)) return false;
        }

        return true;
    }
}